test/test_global_qbx.py

```diff
@@ -257,21 +257,14 @@ def check_quad_res_to_helmholtz_k_ratio(element):
         return quad_res[element.index] * helmholtz_k <= 5
 
     for element_1 in iter_elements(stage1_density_discr):
-        success = True
         for element_2 in iter_elements(stage1_density_discr):
-            result = check_disk_undisturbed_by_sources(element_1, element_2)
-            success = success and result
-        assert success, "'check_disk_undisturbed_by_sources' failed"
+            assert check_disk_undisturbed_by_sources(element_1, element_2)
 
-        success = True
         for element_2 in iter_elements(quad_stage2_density_discr):
-            result = check_sufficient_quadrature_resolution(element_1, element_2)
-            success = success and result
-        assert success, "'check_sufficient_source_quadrature_resolution' failed"
+            assert check_sufficient_quadrature_resolution(element_1, element_2)
 
         if helmholtz_k is not None:
-            success = check_quad_res_to_helmholtz_k_ratio(element_1)
-            assert success, "'check_quad_res_to_helmholtz_k_ratio' failed"
```

Comment:
These are equivalent, right? I have no idea why they were written in this contrived way.

Reply:
Hm, the original version would print out all the problematic elements before asserting, but otherwise they should be the same, yeah.

Reply:
Hmm, OK. I think I'll drop this change then.